gsoc.yml

```diff
@@ -15,7 +15,7 @@ gsoc2023:
     rss_feed: "https://dev.to/feed/pupperemeritus"
     project: 'stingray'
   menasrac:
-    rss_feed: #FIXME
+    rss_feed: "https://medium.com/@menasria6.racim/blog-gsoc-project-common-api-for-large-molecular-database-radis-10c7a1c786e4"
```

Comment:
please provide the link to the RSS feed rather than an individual blogpost. Thanks!